Dockerfile.gsutil

```diff
@@ -11,12 +11,21 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-FROM golang:1.16 as app
-COPY . /go/src/github.com/ossf/scorecard
-WORKDIR /go/src/github.com/ossf/scorecard
-RUN [ "go", "build", "."]
+
+# syntax = docker/dockerfile:1-experimental
+
+FROM  golang:1.16 as base
+WORKDIR /src
+ENV CGO_ENABLED=0
+COPY go.* ./
```

Comment:
Is this needed since line 22 copies . completely ?

Reply:
I tried that it fails with `go mod download: no modules specified (see 'go help mod download')`, so it looks like it is required.

Reply:
Thanks for explaining.